Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby '2.3.0'
+ruby '2.4.0'
```

Comment:
This part may also break some stuff. The unit tests showed up clean, but there may be some edge cases that can't be tested without the production environment.